apps/locale/en_QQ/chromium.app

```diff
@@ -1,11 +1,12 @@
 {
-    "launch_command": "chromium-browser --start-maximized",
+    "launch_command": "kano-tracker-ctl session run 'chromium-browser' 'chromium-browser --start-maximized'",
```

Comment:
The `launch_command` isn't necessary for any `.app` file in the `locale` directory - they will get it from the english version so you can avoid a lot of duplication by removing it.